feature/bgp/policybase/otg_tests/import_export_multi_test/import_export_multi_test.go

```diff
@@ -346,7 +364,9 @@ func configureImportExportMultifacetMatchActionsBGPPolicy(t *testing.T, dut *ond
 		}
 	}
 	communitySetMatchCommPrefixAddCommu.SetCommunityMember(cs4)
-	communitySetMatchCommPrefixAddCommu.SetMatchSetOptions(matchAny)
+	if deviations.BGPConditionsMatchCommunitySetUnsupported(dut) {
```

Comment:
See above, this doesn't appear to do what the intent of the deviation is as written.

Reply:
Thanks for the reference, now I get it!